producers-commissioning/src/unit-test/java/uk/gov/moj/sdt/producers/comx/sdtws/SdtEndpointPortTypeTest.java

```diff
@@ -73,185 +78,163 @@ public class SdtEndpointPortTypeTest extends AbstractSdtUnitTestBase
+        when(mockBulkRequestHandler.getBulkFeedback(any(BulkFeedbackRequestType.class)))
+                .thenThrow (new RuntimeException("test"));
+
+        try {
+            portType.getBulkFeedback (createBulkFeedbackRequestType());
+            fail (RUNTIME_EXCEPTION_SHOULD_HAVE_BEEN_THROWN);
+        } catch (final RuntimeException re) {
+            assertEquals (SDT_SYSTEM_COMPONENT_ERROR, re.getMessage(), "");
         }
 
-        EasyMock.verify (mockBulkRequestHandler);
+        verify (mockBulkRequestHandler).getBulkFeedback(any(BulkFeedbackRequestType.class));
     }
 
     /**
      * Test submit query method completes successfully.
      */
     @Test
-    public void testSubmitQuerySuccess ()
-    {
-        EasyMock.expect (mockSubmitQueryHandler.submitQuery (EasyMock.anyObject (SubmitQueryRequestType.class)))
-                .andReturn (createSubmitQueryResponse ());
-        EasyMock.replay (mockSubmitQueryHandler);
+    void testSubmitQuerySuccess() {
+        when(mockSubmitQueryHandler.submitQuery(any(SubmitQueryRequestType.class)))
+                .thenReturn(createSubmitQueryResponse());
 
-        final SubmitQueryResponseType response = portType.submitQuery (createsubmitQueryRequestType ());
+        final SubmitQueryResponseType response = portType.submitQuery (createsubmitQueryRequestType());
 
-        assertNotNull ("Response expected", response);
-        assertEquals (SDT_COMX_SERVICE, response.getSdtService ());
-        EasyMock.verify (mockSubmitQueryHandler);
+        assertNotNull(response, RESPONSE_EXPECTED);
+        assertEquals(SDT_COMX_SERVICE, response.getSdtService());
+        verify(mockSubmitQueryHandler).submitQuery(any(SubmitQueryRequestType.class));
     }
 
     /**
      * Test submit query method handles exceptions successfully.
      */
     @Test
-    public void testSubmitQueryException ()
-    {
-        EasyMock.expect (mockSubmitQueryHandler.submitQuery (EasyMock.anyObject (SubmitQueryRequestType.class)))
-                .andThrow (new RuntimeException ("test"));
-        EasyMock.replay (mockSubmitQueryHandler);
-
-        try
-        {
-            portType.submitQuery (createsubmitQueryRequestType ());
-            fail ("Runtime exception should have been thrown");
-        }
-        catch (final RuntimeException re)
-        {
-            assertEquals ("",
-                    "A SDT system component error has occurred. Please contact the SDT support team for assistance",
-                    re.getMessage ());
+    void testSubmitQueryException() {
+        when(mockSubmitQueryHandler.submitQuery(any(SubmitQueryRequestType.class)))
+                .thenThrow(new RuntimeException ("test"));
+
+        try {
+            portType.submitQuery (createsubmitQueryRequestType());
+            fail (RUNTIME_EXCEPTION_SHOULD_HAVE_BEEN_THROWN);
+        } catch (final RuntimeException re) {
+            assertEquals(SDT_SYSTEM_COMPONENT_ERROR, re.getMessage(), "");

```

Comment:
Might as well use version of assertEquals() that doesn't have the message parameter.